index.html

```diff
@@ -199,32 +199,34 @@ <h3>Supported sources</h3>
     <h3>Sampling and Reporting Rate</h3>
     <p>
       The <dfn>sampling rate</dfn> for a [=platform collector=] is defined as a rate
-      at which the [=user agent=] obtains telemetry readings from the underlying platform,
-      and it might differ from the pressure observers' [=requested sampling rates=].
```

Comment:
What if instead of removing the "requested sampling rate" dfn, you define both in this section and explain how requested sampling rate and interval can be derived from each other?

This allows you to change fewer places in the spec (since both terms can then be used interchangeably) while allowing you to deal with intervals where it matters (in the internal slot and in the "passes rate test" check).